bodhi-server/bodhi/server/templates/update.html

```diff
@@ -827,7 +842,7 @@ <h3 class="modal-title" id="waiveModalLabel">Trigger Tests</h3>
           icon_class = icons['PASSED'];
         }
         var summary = [];
-        failed_reqs_counter = unsatisfied_reqs_counter - missing_reqs_counter - running_reqs_counter;
+        failed_reqs_counter = unsatisfied_reqs_counter - missing_reqs_counter - running_reqs_counter - remoteerror_reqs_counter;
         if (reqs_counter == 0) {
```

Comment:
This now throws a ReferenceError. I suppose it should be changed to `test_reqs_counter`?
There are some `reqs_counter` references also in the below else/if statements to be changed.